ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py

```diff
@@ -15,21 +15,28 @@
 """
 import json
 import traceback
-from typing import Generic, Optional, Type, TypeVar, Union
+from typing import Dict, Generic, Optional, Type, TypeVar, Union
 
 from pydantic import BaseModel
 
 from metadata.generated.schema.entity.data.table import Table
 from metadata.generated.schema.type import basic
+from metadata.generated.schema.type.entityReference import EntityReference
 from metadata.generated.schema.type.tagLabel import LabelType, State, TagSource
 from metadata.ingestion.ometa.client import REST
 from metadata.ingestion.ometa.utils import model_str
 from metadata.utils.helpers import find_column_in_table_with_index
 from metadata.utils.logger import ometa_logger
 
+try:
+    from typing import get_args
+except ImportError:
+    from typing_compat import get_args
+
 logger = ometa_logger()
 
 T = TypeVar("T", bound=BaseModel)
+U = TypeVar("U", bound=BaseModel)
```

Comment:
also, this one, are we using this TypeVar anywhere? if not then can we remove this?

Reply:
@ulixius9 Good catch.  I changed the parameters in the method I added and no longer needed these.  I cannot figure out how to update the code and push to the same branch, so I'm going to close this PR and open a new one with the requested changes.